src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,110 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private LinkedList<Node<K, V>>[] buckets;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the constructor.